packages/voorbeeld-design-tokens/figma/voorbeeld.tokens.json

```diff
@@ -4703,6 +4703,94 @@
       }
     }
   },
+  "components/task-list": {
+    "todo": {
+      "task-list": {
+        "color": {
+          "type": "color",
+          "value": "{utrecht.document.color}"
+        },
+        "marker": {
+          "checked": {
+            "color": {
+              "type": "color",
+              "value": "{voorbeeld.color.white}"
+            },
+            "background-color": {
+              "type": "color",
+              "value": "{voorbeeld.color.green.600}"
+            },
+            "border-color": {
+              "type": "color",
+              "value": "transparent"
+            }
+          },
+          "size": {
+            "value": "{voorbeeld.size.xs}",
+            "type": "sizing"
+          },
+          "icon": {
+            "size": {
+              "value": "{voorbeeld.size.icon.sm}",
+              "type": "sizing"
+            }
+          },
+          "border-width": {
+            "value": "{voorbeeld.border-width.md}",
+            "type": "borderWidth"
+          },
+          "unchecked": {
```

Comment:
Als er geen officiele term is voor "The Opposite", dan zou ik `not-X` doen: `not-checked` in dit geval.